test/Microsoft.AspNetCore.Server.Kestrel.Performance/configs/MsBuildExecutor.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using BenchmarkDotNet.Characteristics;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Extensions;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains;
+using BenchmarkDotNet.Toolchains.DotNetCli;
+using BenchmarkDotNet.Toolchains.Results;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Performance
+{
+    public class MsBuildExecutor : IExecutor
```

Comment:
@adamsitnik Would be glad to collaborate with you to get this more generalized and into BenchmarkDotNet 😄